base/toml_parser.jl

```diff
@@ -657,13 +657,20 @@ end
 #########
 
 function push!!(v::Vector, el)
+    # Since these types are typically non-inferrable, they are a big invalidation risk,
+    # and since it's used by the package-loading infrastructure the cost of invalidation
+    # is high. Therefore, this is written to reduce the "exposed surface area": e.g., rather
+    # than writing `T[el]` we write it as `push!(Vector{T}(undef, 1), el)` so that there
```

Comment:
tiny nit, but `push!(Vector{T}(undef, 1), el)` should be `push!(Vector{T}(undef, 0), el)` or the version in the code, right?